src/main/java/core/basesyntax/strategy/impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction fruitTransaction) {
+        int quantity = fruitTransaction.getQuantity();
+        String fruit = fruitTransaction.getFruit();
+        if (quantity > 0) {
+            Storage.fruitStorage.put(fruit, quantity);
+        } else {
+            throw new RuntimeException("Invalid quantity of fruit" + quantity);
+        }
```

Comment:
Let's create less `if/else`:
```suggestion
        if (quantity <= 0) {
            throw new RuntimeException("Invalid quantity of fruit" + quantity);
        } 
        Stroage....
```

Applicable for all other places.